lib/sles4sap/ipaddr2.pm

```diff
@@ -238,13 +238,15 @@ sub ipaddr2_infra_deploy {
     }
 
     # If required, create on the fly the cloud-init script
+    # socat is only needed for 12sp5
```

Comment:
But the line 249 would install it in all versions, right?

Reply:
yes, or better to say ensure that it is installed. 15sp5 has it preinstalled so this `zypper in socat`  command juts run doing nothing. I decided to implement it like this not to have one more additional setting to say: "force installing socat".